next/components/atoms/Radio/RadioBox.tsx

```diff
@@ -1,17 +1,18 @@
+import { InfoIcon } from '@assets/icons'
+import Tooltip from '@components/atoms/Tooltip'
 import { RadioGroup } from '@headlessui/react'
 import cx from 'classnames'
-import React, { ReactNode } from 'react'
+import React, { PropsWithChildren, ReactNode } from 'react'
 
 import RadioCircle from './RadioCircle'
 
-/**
- * TODO: Add tooltip when available.
- */
 const RadioBox = ({
   children,
   className,
+  tooltip,
   ...props
-}: Parameters<typeof RadioGroup.Option>[0] & { className?: string; children?: ReactNode }) => {
+}: Parameters<typeof RadioGroup.Option>[0] &
+  PropsWithChildren<{ className?: string; tooltip?: ReactNode }>) => {
```

Comment:
I would prefer to define `RadioBoxProps` type separately, just to be consistent in coding style :)